thunder/benchmarks/__init__.py

```diff
@@ -2958,3 +2960,185 @@ def fn(self) -> Callable:
 #     if args.listbenchmarks:
 #         list_benchmarks(use_classname=False)
 #         sys.exit(0)
+
+from thunder.dynamo.utils import _concrete_shape
+from thunder.dynamo.compiler import ThunderCompiler
+from itertools import chain
+
+GRAPH_BY_GRAPH_BENCHMARK_PARAMS_KEYS = ("GraphID", "SplitModuleName", "executor")
+
+
+def _get_example_inputs_from_placeholder(node) -> tuple[torch.Tensor]:
+    check(node.op == "placeholder", lambda: f"The node must be placeholder type", ValueError)
+    check("example_value" in node.meta, lambda: "example_value does not exist in the meta of {node}", ValueError)
+    example_value = node.meta["example_value"]
```

Comment:
Does it also store strides? If yes let's use it when making tensors below.

Reply:
make_tensor probably can use the memory_format and noncontiguous